src/platforms/blackpillv2/platform.c

```diff
@@ -128,6 +133,8 @@ void platform_request_boot(void)
 	uint32_t *magic = (uint32_t *)&_ebss;
 	magic[0] = BOOTMAGIC0;
 	magic[1] = BOOTMAGIC1;
+	/* Reset core to enter bootloader */
```

Comment:
We would prefer the blank line to be above this comment, not below

Reply:
Will do better next time! Everything has changed a lot since this pr was sent. Ill hafta check if adc support has been brought in for various f4 series etc and a few other areas I had added and if anything is missing that I have would a pr attempt for such things be welcome?

Reply:
You would be welcome to update this PR in that case to reduce the work on both of us